contrib/cirrus/setup_environment.sh

```diff
@@ -100,6 +100,21 @@ esac
 # shellcheck disable=SC2154
 printf "[engine]\ndatabase_backend=\"$CI_DESIRED_DATABASE\"\n" > /etc/containers/containers.conf.d/92-db.conf
 
+# For debian envs pre-configure storage driver as overlay.
+# See: Discussion here https://github.com/containers/podman/pull/18510#discussion_r1189812306
+# for more details.
+# TODO: remove this once all CI VM have newer buildah version. (i.e where buildah
+# does not defaults to using `vfs` as storage driver)
+buildah_storage=$(buildah info --format '{{.store.GraphDriverName}}')
```

Comment:
When run as root, this always returns `overlay`. I'm sorry, I didn't know that. I guess it's OK to revert to checking for "debian", unless someone can think of a better test.